search_that_hash/api.py

```diff
@@ -1,26 +1,25 @@
-from search_that_hash.cracker import cracking
 from search_that_hash import config_object
 from search_that_hash import __main__
-
+from search_that_hash.cracker import handler
 import json
 
 
 def return_as_json(hashes):
 
     config = config_object.api_config(hashes)
     config["greppable"] = True
+    cracking_handler = handler.main(config)
+
+    x = cracking_handler.start()
 
-    searcher = cracking.Searcher(config)
-    x = searcher.main()
-    print(x)
     return x
 
 
+def return_as_fast_json(hashes):
 
-def return_as_fast_json(hashes, sth_api):
+    config = config_object.api_config(hashes)
+    cracking_handler = handler.main(config)
 
-    config = config_object.api_config(hashes, sth_api)
-    searcher = cracking.Searcher(config)
-    x = searcher.main()
+    x = cracking_handler.start()
```

Comment:
change var name, perhaps to `cracking_result`?

Also, what is Fast json? I have asked this repeatedly, I have literally no idea what it does. It's not in the documentation.

If you cannot explain what it does, remove this whole function as it's not needed :)